java/src/test/java/org/rocksdb/DBOptionsTest.java

```diff
@@ -8,8 +8,8 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 
-import java.util.Properties;
-import java.util.Random;
+import java.nio.file.Paths;
+import java.util.*;
```

Comment:
Yup. However, IntelliJ automatically rewrites those to a `.*` for you once you get past a few.